### DISCUSSION TITLE
Colombia Central (Bogota)

### DISCUSSION THREAD

**Discussion by javierbm7** [VarunAgw/CloudPingTest#27] [Ideas] [answered]
Hi can you add Colombia Central (Bogota)  sa-bogota-1

**Comment by CarlosBiscione** [comment]
Hi, Varun...
Yes... Oracle Cloud first Colombia Region is now live for the world, but still missing in your wonderful tool.
Is it possible to get it listed on there?
Thanks in advance.

**Comment by VarunAgw** [comment, marked as answer]
Thanks for the reminder. I have added them
@CarlosBiscione  @javierbm7 

https://cloudpingtest.com/oracle

**Comment by ocicloudengineer** [comment]
Certainly, the Colombia region is now available, and it would be wonderful to have a way to showcase this feature to individuals who wish to use the region.

**Comment by VarunAgw** [comment]
@ocicloudengineer Please make a new discussion for such ideas
